cirq-google/cirq_google/experimental/noise_models/validate_noise_model_from_calibration.py

```diff
@@ -0,0 +1,85 @@
+import cirq_google
+import numpy as np
+from cirq.devices.noise_properties import NoiseModelFromNoiseProperties
+from cirq_google.experimental.noise_models.calibration_to_noise_properties import (
+    noise_properties_from_calibration,
+)
+import cirq
+import pandas as pd
+
+
+def validate_noise_model(
```

Comment:
"validate" doesn't seem right here, since we're not returning an error. Maybe "compare_generated_noise_to_metrics"?